doc/manual/local.mk

```diff
@@ -13,15 +17,24 @@ XSLTPROC = $(xsltproc) --nonet $(xmlflags) \
   --stringparam generate.toc "book toc" \
   --param keep.relative.image.uris 0
 
-docbookxsl = http://docbook.sourceforge.net/release/xsl-ns/current
-docbookrng = http://docbook.org/xml/5.0/rng/docbook.rng
+ifeq ($(NONET),0)
+    docbookxsl = /usr/share/xml/docbook/stylesheet/docbook-xsl
+    docbookrng = $(docbookxsl)/slides/schema/relaxng/docbook.rng
+    profile_xsl= $(docbookxsl)/profiling/profile.xsl
+    docbook_xsl= $(docbookxsl)/manpages/profile-docbook.xsl
+else
+    docbookrng = "http://docbook.org/xml/5.0/rng/docbook.rng"
+    docbookxsl = "http://docbook.sourceforge.net/release/xsl-ns/current"
+    profile_xsl= $(docbookxsl)/profiling/profile.xsl
+    docbook_xsl= $(docbookxsl)/manpages/docbook.xsl
+endif
```

Comment:
Yes, it is orthogonal. 
It is/was _just_ to get the documentation to work. 
I should at least split it out into a separate commit, and possibly setup _another_ configuration variable to handle where **docbook-xsl** (and other dependencies) might be installed (if at all).
Otherwise, downloading anything during the build (by default) should be forbidden (perhaps we should be forced to turn it on explicitly, beforehand).